scripts/scil_volume_pairwise_comparison.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Evaluate pair-wise similarity measures of masks and atlas.
+All volumes must be co-registered in the same space.
+
+Support multiple input volume. The following command will compare all
+combinations of the input volumes (1-2, 1-3, 2-3):
+  scil_volume_pairwise_comparison.py mask1.nii.gz mask2.nii.gz \
+    mask3.nii.gz out.json
+
+The following command will compare all input of the input volumes to a single
+volume (1-ref, 2-ref, 3-ref):
+  scil_volume_pairwise_comparison.py mask1.nii.gz mask2.nii.gz \
+    mask3.nii.gz out.json --single_compare ref.nii.gz
+
+This can work for BET mask, WMPARC, bundle label maps. The datatype of the
+input volumes must be uint8 (mask) or uint16 (label map and atlas).
+The computed similarity measures are:
+    adjacency_voxels, dice_voxels, volume_overlap, volume_overreach.
+For each measure, an entry in the json file will be created and for each unique
+value present in the input volumes there will be an entry under the measure.
+(i.e. a binary mask will have one entries for each measure, 1).
+
+If you have streamlines to compare, the following script could be
+of interest for you: scil_bundle_pairwise_comparison.py
+"""
+
+
+import argparse
+import itertools
+import json
+import logging
+import multiprocessing
+
+from dipy.io.utils import get_reference_info
+import numpy as np
+
+from scilpy.io.image import load_img
+from scilpy.io.utils import (add_json_args,
+                             add_overwrite_arg,
+                             add_processes_arg,
+                             add_verbose_arg,
+                             assert_headers_compatible,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             validate_nbr_processes)
+from scilpy.image.labels import get_data_as_labels
+from scilpy.tractanalysis.reproducibility_measures import compare_volume_wrapper
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('in_volumes', nargs='+',
```

Comment:
I think you should put `nargs='*' ` which is 2 or more volumes. You don't want only one volume.